src/components/invoice/InvoiceSendBill.tsx

```diff
@@ -0,0 +1,158 @@
+import type { Invoice, User } from "@prisma/client";
+import type { Dispatch, SetStateAction } from "react";
+import React, { useEffect, useState } from "react";
+import type { SubmitHandler } from "react-hook-form";
+import { useForm } from "react-hook-form";
+
+import type { InvoiceRowData } from "../../pages/invoice/index";
+import type { RouterInputs, RouterOutputs } from "../../utils/api";
+import { api } from "../../utils/api";
+import Alert from "../Alert";
+import type { TablePopup } from "../tables/input";
+
+import type { InvoicePopupTypes } from "./InvoicePopup";
+
+const patients: User[] = [];
+/**
+ * instead of displaying all hours, grab the available times for the current doctor
+ */
+const getPatients = async function () {
+  try {
+    const body = {};
+    await fetch(`/api/getPatients`, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(body),
+    })
+      .then((response) => response.json())
+      .then((pats: User[]) => {
+        patients.length = 0;
+        pats.forEach((pat) => {
+          patients.push(pat);
+        });
+      });
+  } catch (error) {
+    console.error(error);
+  }
+};
+const patGetter = getPatients();
```

Comment:
I think I've seen this function several times now, maybe it could be moved to utils/ to be reused across all the files that need it?